packages/bundle-size/src/utils/collectLocalReport.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs').promises;
+const glob = require('glob');
+const path = require('path');
+const { findGitRoot, findPackageRoot } = require('workspace-tools');
+
+/** @typedef {import('../utils/buildFixture').BuildResult} BuildResult */
+
+/** @typedef {BuildResult & { packageName: string }} BundleSizeReportEntry */
+/** @typedef {BundleSizeReportEntry[]} BundleSizeReport */
+
+/**
+ * @param {string} reportFile
+ *
+ * @return {Promise<[string, BuildResult[]]>}
+ */
+async function readReportForPackage(reportFile) {
+  const reportFilePath = path.resolve(process.cwd(), reportFile);
+
+  const packageName = path.basename(/** @type {string}*/ (findPackageRoot(reportFilePath)));
```

Comment:
can we properly handle this 3rd party call? (type overrides are ususaly a code smell)

```
const packageRoot = findPackageRoot(reportFilePath);

  if (!packageRoot) {
    throw new Error('.....');
  }

const packageName = path.basename(packageRoot);
```

Reply:
Handled in a85a50d.